linux_pipeline/Jenkinsfile_container_images

```diff
@@ -226,5 +227,20 @@ pipeline {
           }
         }
       }
+      stage('Publish artifacts') {
```

Comment:
change the stage name to publish_artifacts (to be consistent with the other naming)